cmd/local-artifact-mirror/pull_binaries.go

```diff
@@ -29,22 +56,54 @@ func PullBinariesCmd(cli *CLI) *cobra.Command {
 				return fmt.Errorf("unable to create kube client: %w", err)
 			}
 
-			in, err := fetchAndValidateInstallation(ctx, kcli, args[0])
+			isAirgap := licenseID == ""
```

Comment:
why isn't this just driven by `installation.Spec.AirGap`? using the license id makes it so we have to ensure both variables are taken into account when we use this command.

Reply:
but you don't need to. the installation is the source of truth. it should simply be updated to not fail if it's not airgap.

Reply:
Updated https://github.com/replicatedhq/embedded-cluster/pull/2122/commits/81bc5d5fa42d879dcb6df77e14da71a3feca78f6